definitions/reports/virt_who.rb

```diff
@@ -0,0 +1,21 @@
+module Reports
+  class Virtwho < ForemanMaintain::Report
+    metadata do
+      description 'Check if virt-who is being used and what hypervisor types are present'
+    end
+
+    HYPERVISOR_TYPES = %w[ahv esx fakevirt hyperv kubevirt libvirtd].freeze
+
+    def run
+      self.data = {}
```

Comment:
```suggestion
```

`data_field` will do that for us

Reply:
The `each` statement still uses it.
I didn't have the time to think about how to pass context parameters to the data_field/merge_data methods.